addon/session-stores/cookie.js

```diff
@@ -159,6 +159,11 @@ export default BaseStore.extend({
   init() {
     this._super(...arguments);
 
+    if (!this.get('cookieExpirationTime')) {
```

Comment:
I think we actually always want to override `cookieExpirationTime`. The idea here is that if you define your application session store as

```js
export default Cookie.extend({
  cookieExpirationTime: 60 * 60
});
```

but then have a remember-me action in the UI that updates the expiration time like

```js
rememberMe() {
  this.set('session.store.cookieExpirationTime', 60 * 60 * 24);
}
```

you'll want that value of `60 * 60 * 24` to be applied after the next reload although the initial value of `60 * 60` would be set in this case as well.